### PR TITLE
[Bug fix] LPS-129172 Upgrade jackson

### PR DESCRIPTION
**Pull request by yuhai**
@tinatian, the pr is based on https://github.com/tinatian/liferay-portal/pull/2330#issuecomment-896447577
cc @dantewang

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-cart-taglib/build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dependencies {
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.10.3"
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.10.3"
-	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-databind", version: "2.10.5.1"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.12.4"
```
Comment (tinatian) [inline modules/apps/commerce/commerce-cart-taglib/build.gradle:2]: Hi @yuhai @dantewang
Let's try to get ride of the compileInclude usage like this, only use compileInclude when you are the only one need this jar.
jackson related jars are used in many places, let's move it to lib/portal and export the packages like this https://github.com/shuyangzhou/liferay-portal/pull/10218/commits/abfa60395a5000d754172bbced581bb23b499a2b

**Comment by tinatian** [comment]
@yuhai @dantewang Please see the comment above and resend.

Thanks.
Tina

**Pull request closed without merging**